src/main/java/se/sundsvall/casedata/api/model/Stakeholder.java

```diff
@@ -41,6 +38,14 @@ public class Stakeholder {
 	@Schema(accessMode = Schema.AccessMode.READ_ONLY, description = "The version of the stakeholder", example = "1")
 	private int version;
 
+	@Schema(description = "The municipality ID", example = "2281", accessMode = Schema.AccessMode.READ_ONLY)
+	@Size(max = 255)
```

Comment:
Validation constraint not necessary since this is a readonly-property

Comment applicable for all READ_ONLY-fields (for municipalityId) in other API-model classes.